Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
+
+COPY . /app
+WORKDIR /app
+RUN pip install -e .
+
+ENV XSNIPPET_API_SETTINGS=/app/xsnippet_api/default.conf
```

Comment:
If I'm not mistaken, `XSNIPPET_API_SETTINGS` should point to an additional external config. Default config will be used automatically, no need to point on it.